<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeilcalMainWindow</class>
 <widget class="QMainWindow" name="HeilcalMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HeilcalMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>211</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>SSH Session</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>149</y>
       <width>82</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="terminalButton">
        <property name="text">
         <string>Terminal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="executeCommandButton">
        <property name="text">
         <string>Execute</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sftpButton">
        <property name="text">
         <string>SFTP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnectServerButton">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>96</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="serverLabel">
          <property name="text">
           <string>Server:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="serverNameLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="userLabel">
          <property name="text">
           <string>User:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="userNameLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="authorizationLabel">
          <property name="text">
           <string>Authorization:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="authorizationTypeLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentStatusLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextEdit" name="serverSessionLog">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>441</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnections"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionConnections">
   <property name="text">
    <string>Connections</string>
   </property>
   <property name="toolTip">
    <string>Connections</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
